packages/react-redux-subspace/src/components/SubspaceProvider.js

```diff
@@ -25,22 +58,40 @@ class SubspaceProvider extends React.PureComponent {
     }
 }
 
+SubspaceLegacyContextOutput.childContextTypes = {
+    store: PropTypes.object
+}
+
+class SubspaceProvider extends React.Component {
+    makeSubspace = memoizeOne((mapState, namespace, subspaceDecorator, store) => {
+        const createSubspace = subspaceDecorator || subspace(mapState, namespace)
+        return createSubspace(store)
+    })
+
+    render() {
+        const { mapState, namespace, subspaceDecorator, store, children } = this.props
+        const subspace = this.makeSubspace(mapState, namespace, subspaceDecorator, store)
+        return (
+            <Provider key={subspace.namespace} store={subspace}>
+                <SubspaceLegacyContextOutput subspace={subspace}>
```

Comment:
This will cause the subspace to always be put into legacy context for and downstream `SubspaceProviders` or `dynostore` `dynamic` components using older versions to access.

Reply:
Definately a necessary evil.